portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,30 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Miguel Fabrigas</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="crt">
+      <p>> Welcome!<span id="cursor">|</span></p>
```

Comment:
Add a space between `<p>` and `>` to make it more readable

Reply:
Fixed